llm_bioinformatics_research/src/LoginScreen/LoginScreen.js

```diff
@@ -130,7 +129,7 @@ function LoginScreen() {
             </Link>
```

Comment:
Would it be possible to add the option of entering your username as well so if the user doesn't remember the email they used or vice versa, they are still able to login?